{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_vex",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2023 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "SUSE CVE-2019-19275",
				"title": "Title"
			},
			{
				"category": "description",
				"text": "typed_ast 1.3.0 and 1.3.1 has an ast_for_arguments out-of-bounds read. An attacker with the ability to cause a Python interpreter to parse Python source (but not necessarily execute it) may be able to crash the interpreter process. This could be a concern, for example, in a web-based service that parses (but does not execute) Python code. (This issue also affected certain Python 3.8.0-alpha prereleases.)",
				"title": "Description of the CVE"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "CVE-2019-19275",
				"url": "https://www.suse.com/security/cve/CVE-2019-19275"
			},
			{
				"category": "external",
				"summary": "SUSE Security Ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "external",
				"summary": "SUSE Bug 1161563 for CVE-2019-19275",
				"url": "https://bugzilla.suse.com/1161563"
			},
			{
				"category": "external",
				"summary": "Advisory link for openSUSE-SU-2020:0567-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/YRRUBXXZHPQ4KOXSBUH6PB74WVDSJHI5/"
			},
			{
				"category": "external",
				"summary": "Advisory link for openSUSE-SU-2020:0609-1",
				"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/TUHDFY53Y3VOY7VK7BOC2B4H5HIPQAFJ/"
			}
		],
		"title": "SUSE CVE CVE-2019-19275",
		"tracking": {
			"current_release_date": "2023-02-15T04:06:17Z",
			"generator": {
				"date": "2023-02-15T04:06:17Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf-vex.pl",
					"version": "1"
				}
			},
			"id": "CVE-2019-19275",
			"initial_release_date": "2023-02-15T04:06:17Z",
			"revision_history": [
				{
					"date": "2023-02-15T04:06:17Z",
					"number": "2",
					"summary": "Current version"
				}
			],
			"status": "interim",
			"version": "2"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Package Hub 15 SP1",
								"product": {
									"name": "SUSE Package Hub 15 SP1",
									"product_id": "SUSE Package Hub 15 SP1"
								}
							},
							{
								"category": "product_name",
								"name": "openSUSE Leap 15.1",
								"product": {
									"name": "openSUSE Leap 15.1",
									"product_id": "openSUSE Leap 15.1",
									"product_identification_helper": {
										"cpe": "cpe:/o:opensuse:leap:15.1"
									}
								}
							},
							{
								"category": "product_version",
								"name": "python3-typed-ast-1.3.1-bp151.2.6.1",
								"product": {
									"name": "python3-typed-ast-1.3.1-bp151.2.6.1",
									"product_id": "python3-typed-ast-1.3.1-bp151.2.6.1"
								}
							},
							{
								"category": "product_version",
								"name": "python3-typed-ast-1.3.1-lp151.2.6.1",
								"product": {
									"name": "python3-typed-ast-1.3.1-lp151.2.6.1",
									"product_id": "python3-typed-ast-1.3.1-lp151.2.6.1"
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-typed-ast-1.3.1-bp151.2.6.1 as component of SUSE Package Hub 15 SP1",
					"product_id": "SUSE Package Hub 15 SP1:python3-typed-ast-1.3.1-bp151.2.6.1"
				},
				"product_reference": "python3-typed-ast-1.3.1-bp151.2.6.1",
				"relates_to_product_reference": "SUSE Package Hub 15 SP1"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python3-typed-ast-1.3.1-lp151.2.6.1 as component of openSUSE Leap 15.1",
					"product_id": "openSUSE Leap 15.1:python3-typed-ast-1.3.1-lp151.2.6.1"
				},
				"product_reference": "python3-typed-ast-1.3.1-lp151.2.6.1",
				"relates_to_product_reference": "openSUSE Leap 15.1"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2019-19275",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-19275"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "typed_ast 1.3.0 and 1.3.1 has an ast_for_arguments out-of-bounds read. An attacker with the ability to cause a Python interpreter to parse Python source (but not necessarily execute it) may be able to crash the interpreter process. This could be a concern, for example, in a web-based service that parses (but does not execute) Python code. (This issue also affected certain Python 3.8.0-alpha prereleases.)",
					"title": "Vulnerability description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE Package Hub 15 SP1:python3-typed-ast-1.3.1-bp151.2.6.1",
					"openSUSE Leap 15.1:python3-typed-ast-1.3.1-lp151.2.6.1"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-19275",
					"url": "https://www.suse.com/security/cve/CVE-2019-19275"
				},
				{
					"category": "external",
					"summary": "SUSE Security Ratings",
					"url": "https://www.suse.com/support/security/rating/"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1161563 for CVE-2019-19275",
					"url": "https://bugzilla.suse.com/1161563"
				},
				{
					"category": "external",
					"summary": "Advisory link for openSUSE-SU-2020:0567-1",
					"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/YRRUBXXZHPQ4KOXSBUH6PB74WVDSJHI5/"
				},
				{
					"category": "external",
					"summary": "Advisory link for openSUSE-SU-2020:0609-1",
					"url": "https://lists.opensuse.org/archives/list/security-announce@lists.opensuse.org/thread/TUHDFY53Y3VOY7VK7BOC2B4H5HIPQAFJ/"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Package Hub 15 SP1:python3-typed-ast-1.3.1-bp151.2.6.1",
						"openSUSE Leap 15.1:python3-typed-ast-1.3.1-lp151.2.6.1"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7.5,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
						"version": "3.1"
					},
					"products": [
						"SUSE Package Hub 15 SP1:python3-typed-ast-1.3.1-bp151.2.6.1",
						"openSUSE Leap 15.1:python3-typed-ast-1.3.1-lp151.2.6.1"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2019-11-26T19:58:51Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2019-19275"
		}
	]
}
